import sys
try:
    11/0
    print(x)
except Exception as e:
    print(e)
